### PR TITLE
Ajouter les établissements à l'index Elasticsearch

### LINKED ISSUE #70: Ajouter les données des établissements au workflow

### PR DESCRIPTION
**Pull request by HAEKADI**

### CODE REVIEW

**Comment by HAEKADI** [comment]
I am going to remove the `convention_collective` files once @geoffreyaldebert  is done testing !

File: DAG-insert-elk-sirene.py [hunk 1]
```diff
@@ -50,7 +51,7 @@
     default_args=default_args,
     schedule_interval="0 23 10 * *",
     start_date=days_ago(10),
-    dagrun_timeout=timedelta(minutes=60 * 8),
+    dagrun_timeout=timedelta(minutes=60 * 20),
```
Comment (geoffreyaldebert) [inline DAG-insert-elk-sirene.py:54]: Hum. Can we go back to 8 hours max please

File: DAG-update-elk-convcollective.py [hunk 2]
```diff
@@ -1,128 +0,0 @@
-from datetime import timedelta
```
Comment (geoffreyaldebert) [inline DAG-update-elk-convcollective.py:1]: Can we keep this file. Update in target architecture can be adapted but not suppressed entirely. I would like to have a backbone (like this one) to start reshaping it.

File: data_aggregation/convention_collective.py [hunk 3]
```diff
@@ -1,47 +0,0 @@
-import os
```
Comment (geoffreyaldebert) [inline data_aggregation/convention_collective.py:1]: Same here. Can be used for starting point for later dev on updates.

File: elasticsearch/mapping_sirene_index.py [hunk 4]
```diff
@@ -94,79 +152,42 @@ class ElasticsearchSireneIndex(Document):
...
-    type_voie = Text()
-    tranche_effectif_salarie_siege = Keyword()
+    siret_siege = Keyword()
     tranche_effectif_salarie_unite_legale = Keyword()
 
     class Index:
         name = f"siren-{NEXT_COLOR}"
         settings = {
-            "number_of_shards": 1,
+            "number_of_shards": 4,
```
Comment (geoffreyaldebert) [inline elasticsearch/mapping_sirene_index.py:189]: this could maybe be a variable ?

**Review by geoffreyaldebert — APPROVED** [review]
👏

File: DAG-insert-elk-sirene.py (hunk 1 above)
Comment (HAEKADI) [reply DAG-insert-elk-sirene.py:54]: Yes :)

File: DAG-update-elk-convcollective.py (hunk 2 above)
Comment (HAEKADI) [reply DAG-update-elk-convcollective.py]: Okay, done !

File: elasticsearch/mapping_sirene_index.py (hunk 4 above)
Comment (HAEKADI) [reply elasticsearch/mapping_sirene_index.py:189]: Yes !

Comment (HAEKADI) [reply elasticsearch/mapping_sirene_index.py]: I added it in the same file for now. It's complicated to add it to the class through a parameter.

**Pull request closed without merging**